.github/workflows/haskell.yml

```diff
@@ -12,6 +12,9 @@ jobs:
       - uses: haskell/actions/setup@v2
         with:
           enable-stack: true
+      - if: ${{ runner.os == 'Linux' }}
```

Comment:
Are we able to drop the changes to this file?

Reply:
https://github.com/actions/runner-images/issues/7061 seems already been solved. Right!